website/client/js/services/costumeServices.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+(function(){
+  angular
+    .module('habitrpg')
+    .factory('Costume', costumeFactory);
+
+  costumeFactory.$inject = [
+    'Content'
+  ];
+
+  function costumeFactory(Content) {
+    
+    function formatAnimal(name, type) {
+      if(type === 'pet' && name in Content.petsInfo) {
+        return Content.petsInfo[name].text();
+      } else if(type === 'mount' && name in Content.mountsInfo) {
+        return Content.mountsInfo[name].text();
+      }
+    }
+
+    function bgString(set, bg) {
+      var setString = window.env.t(set);
```

Comment:
Weird. I'll have to review the logic again. I thought for sure it was organized by set.